src/libraries/Microsoft.PowerFx.Interpreter/Functions/CommonErrors.cs

```diff
@@ -168,6 +168,16 @@ public static ErrorValue NotYetImplementedError(IRContext irContext, string mess
             });
         }
 
+        public static ErrorValue NotSupportedError(IRContext irContext)
+        {
+            return new ErrorValue(irContext, new ExpressionError()
+            {
+                Message = $"Not supported",
```

Comment:
>Not supported [](http://example.com/codeflow?start=28&length=13)

Shouldn't this error be more "Invalid Operation" as it's used when trying to a number and a date or a number and a time.

Reply:
Updated the message. Also moved this to LibraryOperators, since this is only used in that file anyway.